appveyor.yml

```diff
@@ -34,19 +50,25 @@ pull_requests:
 
 install:
   - cmd: git submodule update --init --recursive
+  - sh: git submodule update --init --recursive
```

Comment:
same for windows and Linux; I tried it in `ps `but it didn't work, so I kept `cmd `for Win and added `sh `for Linux